### DISCUSSION TITLE
Status: 0 | communication failure error [Bug]:  | Pimcore 10.5.0

### DISCUSSION THREAD

**Discussion by GaneshKandu** [pimcore/pimcore#12729] [Q&A / Support]
Hi
I am getting ``communication failure`` error all the time am login in pimcore 

### dev.log

[dev.log](https://github.com/pimcore/pimcore/files/9158873/dev.log)

### screenshot

![image](https://user-images.githubusercontent.com/12833130/180201403-86873d5f-3488-4990-a311-567fde9a40a3.png)

### Steps to reproduce

Installed Pimcore 10.5.0 and login to admin panel

php 8.0.21
mysql 8.0.29

**Comment by dvesh3** [comment]
@GaneshKandu please start with the a discussion, if you are facing any issue. It could be related to your server or cache issue etc. 

If we confirm, it is a valid bug then we can create an issue from the discussion and work on it. thanks! 😊

**Comment by dvesh3** [comment]
Is this skeleton or demo installation?

**Reply by GaneshKandu** [reply]
> Is this skeleton or demo installation?

skeleton

**Reply by dvesh3** [reply]
sorry, I can't reproduce the problem. Please provide reproducible steps to the issue.

**Reply by GaneshKandu** [reply]
```
xhr request to /admin/class/get-tree?createAllowed=true&_dc=1663317786136 failed startup.js:183:21
    <anonymous> https://mydomain.com/bundles/pimcoreadmin/js/pimcore/startup.js?_dc=e47272e09c12046883f680e50571ab53491eac47:183
    ExtJS 26
    <anonymous> https://mydomain.com/bundles/pimcoreadmin/js/pimcore/startup.js?_dc=e47272e09c12046883f680e50571ab53491eac47:443
    ExtJS 47
```

https://github.com/pimcore/pimcore/blob/9fccc1fb2295060a0bde7f29c3355e352a44cd14/bundles/AdminBundle/public/js/pimcore/startup.js#L183

https://github.com/pimcore/pimcore/blob/9fccc1fb2295060a0bde7f29c3355e352a44cd14/bundles/AdminBundle/public/js/pimcore/startup.js#L446

```
xhr request to /admin/settings/get-available-admin-languages?_dc=1663317786138 failed startup.js:183:21
    <anonymous> https://mydomain.com/bundles/pimcoreadmin/js/pimcore/startup.js?_dc=e47272e09c12046883f680e50571ab53491eac47:183
    ExtJS 26
    <anonymous> https://mydomain.com/bundles/pimcoreadmin/js/pimcore/startup.js?_dc=e47272e09c12046883f680e50571ab53491eac47:470
    ExtJS 47
```

https://github.com/pimcore/pimcore/blob/9fccc1fb2295060a0bde7f29c3355e352a44cd14/bundles/AdminBundle/public/js/pimcore/startup.js#L183

https://github.com/pimcore/pimcore/blob/9fccc1fb2295060a0bde7f29c3355e352a44cd14/bundles/AdminBundle/public/js/pimcore/startup.js#L473

```
xhr request to /admin/settings/get-available-sites?_dc=1663317786139 failed startup.js:183:21
    <anonymous> https://mydomain.com/bundles/pimcoreadmin/js/pimcore/startup.js?_dc=e47272e09c12046883f680e50571ab53491eac47:183
    ExtJS 26
    <anonymous> https://mydomain.com/bundles/pimcoreadmin/js/pimcore/startup.js?_dc=e47272e09c12046883f680e50571ab53491eac47:491
    ExtJS 47
```

https://github.com/pimcore/pimcore/blob/9fccc1fb2295060a0bde7f29c3355e352a44cd14/bundles/AdminBundle/public/js/pimcore/startup.js#L183

https://github.com/pimcore/pimcore/blob/9fccc1fb2295060a0bde7f29c3355e352a44cd14/bundles/AdminBundle/public/js/pimcore/startup.js#L494

```
xhr request to /admin/targeting/target-group/list?_dc=1663317786151 failed startup.js:183:21
    <anonymous> https://mydomain.com/bundles/pimcoreadmin/js/pimcore/startup.js?_dc=e47272e09c12046883f680e50571ab53491eac47:183
    ExtJS 26
    <anonymous> https://mydomain.com/bundles/pimcoreadmin/js/pimcore/startup.js?_dc=e47272e09c12046883f680e50571ab53491eac47:511
    ExtJS 47
```

https://github.com/pimcore/pimcore/blob/9fccc1fb2295060a0bde7f29c3355e352a44cd14/bundles/AdminBundle/public/js/pimcore/startup.js#L183

https://github.com/pimcore/pimcore/blob/9fccc1fb2295060a0bde7f29c3355e352a44cd14/bundles/AdminBundle/public/js/pimcore/startup.js#L514

I found this issues related to communication failure 
I pointed all urls directly using browser, all are working but from pimcore ajax call they are failed with NS_BINDING_ABORTED status.
